ga4mp/ga4mp.py

```diff
@@ -343,3 +343,61 @@ def _check_date_not_in_future(self, date):
             assert (
                 date <= datetime.datetime.now()
             ), "Provided date cannot be in the future"
+
+class gtagMP(Ga4mp):
```

Comment:
Capitalize

Reply:
Done, and I updated it in the markdown for the readme, too.